### ISSUE TITLE
enable changefeed queue overflow test

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#5032]
There has been a commented-out test for overflowing the changefeed buffer in `table.yaml` for some time, but it was blocked by #3607. Now that that restriction has been lifted we should re-enable the test.

**Comment by larkost** [comment]
This is up for review with CR 3313 from branch larkost/5032-changefeed-queue-test

**Comment by larkost** [comment]
This has been merged into `next` with 588622dc
